krait-ui/src/actions/fetch-records.ts

```diff
@@ -70,11 +70,17 @@ export default class FetchRecords extends BaseAction<
    * Sets the filters query to the request url using
    * the passed form.
    *
-   * @param {HTMLFormElement} form - The filters form element.
+   * @param {string} formSelector - The query selector for filter form element.
    * @param {URL} url - The request url
    * @private
    */
-  private setFilters(form: HTMLFormElement, url: URL): void {
+  private setFilters(formSelector: string, url: URL): void {
+    const form = document.querySelector<HTMLFormElement>(formSelector);
+
+    if (!form) {
```

Comment:
@Berat-Dzhevdetov,

Let's make the configurations available in all actions. Place them in the `BaseAction` class.